web/client/components/TOC/Toolbar.jsx

```diff
@@ -330,14 +341,14 @@ class Toolbar extends React.Component {
                         </OverlayTrigger>
                         : null}
                     {this.props.activateTool.activateSwipeOnLayer && (status === 'LAYER') &&
-                        <OverlayTrigger
-                            key="layerSwipe"
-                            placement="top"
-                            overlay={<Tooltip id="layer-tooltip-swipe"><Message msgId="toc.compareTool" /></Tooltip>}>
-                            <Button key="layer-swipe" bsStyle={this.props?.swipeSettings?.active ? "success" : "primary"} className="square-button-md" onClick={() => this.showSwipeSettings(status)}>
-                                <Glyphicon glyph="transfer" />
-                            </Button>
-                        </OverlayTrigger>}
+                        <DropdownButtonT
+                            bsStyle={this.props?.swipeSettings?.active ? "success" : "primary"}
+                            noCaret {...swipeToolButtonConfig}>
+                            <MenuItem active={this.props.swipeSettings.mode === "swipe"} onClick={() => this.showSwipeTools(status, "swipe")}>Swipe</MenuItem>
```

Comment:
Please localized tool titles. Use "Swipe" and "Spy glass" for all the languages for the moment